### PR TITLE
add rest of sample files to style checker

### PR DESCRIPTION
**Pull request by vaavva**
## Purpose

* PR adds the rest of our sample files to the style checker so everything is covered instead of only some files from some samples
* resolves https://github.com/Azure-Samples/iot-middleware-freertos-samples/issues/316

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[x] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

**Comment by vaavva** [comment]
I left out any device support code, like `iot-middleware-freertos-samples/demos/projects/ESPRESSIF/adu/components/`, `demos/projects/PC/windows/WinPCap`, and `demos/projects/ST/b-l475e-iot01a/st_code`

File: demos/projects/NXP/mimxrt1060/main.c [hunk 1]
```diff
@@ -527,36 +527,50 @@ int iMainRand32( void )
 }
 /*-----------------------------------------------------------*/
 
+/* *INDENT-OFF* */
```
Comment (vaavva) [inline demos/projects/NXP/mimxrt1060/main.c:530]: Line 530 and 554 are the only manual changes in the demo files, to mitigate a limitation with uncrustify: https://github.com/uncrustify/uncrustify/issues/2032

**Comment by danewalton** [comment]
From https://man.archlinux.org/man/community/uncrustify/uncrustify.1.en#EXAMPLES

Can we utilize something like `uncrustify -c my.cfg --no-backup $(find src -name "*.[ch]")` to simplify this at all? Just because it's getting a bit long 😬

**Review by danewalton — APPROVED** [review]
it's beautiful 🥲

**Pull request merged**